test/parser.jl

```diff
@@ -1009,6 +1010,7 @@ end""" |> test_expr
         @test CSTParser.parse(raw"'\u222ää'").head == :errortoken
         @test CSTParser.parse(raw"'\x222ää'").head == :errortoken
         @test CSTParser.parse(raw"'\U222ää'").head == :errortoken
+        @test CSTParser.parse(raw"'\U10000001'").head == :errortoken
```

Comment:
This is actually a parser error in Julia's own parser:
```jl
julia> '\U10000001'
ERROR: syntax: invalid escape sequence
```
The largest valid code point is U+10FFFF.

Reply:
Yeah, that's basically what this test tests:
```
julia> CSTParser.parse(raw"'\U10000001'")
  1:12  errortoken( CSTParser.InvalidChar)
  1:12   CHAR: '\'

julia> CSTParser.parse(raw"'\U00000001'")
  1:12  CHAR: '\U00000001'
```
Could've chosen a less arbitrary codepoint though, I suppose.

Reply:
Wouldn't it make sense to test the two values on either side? I.e. test that `'\U10FFFF'` is valid and that `'\U110000'` is invalid?